kmk/extensions/display/builtin.py

```diff
@@ -0,0 +1,24 @@
+from kmk.extensions.display import DisplayBackend
+
+
+# Intended for displays with drivers built into CircuitPython
+# that can be used directly without manual initialization
+class backend(DisplayBackend):
```

Comment:
```suggestion
class Backend(DisplayBackend):
```

Class names are CamelCase.
I personally don't 100% agree that all the backend implementations should have the same name. They are in seperate, appropriately named submodules though, so there's that and I'm not going to complain about it.
There could be an argument, that `__init__.DisplayBackend` is a slight misnomer, as it's specifically not a backend, more of a "stub" or "abstract" class, whatever your preferred nomenclature. I'd appreciate your opinion on that.

Reply:
My reasoning for this suggestion was to reduce tautology and make it so that one could intuit the process of setting up a backend from prior experience with another. I'm also not super into calling those backends.

Reply:
I get the reasoning and it's a valid point. The reason I prefer tautologies is that no matter where it turns up, be it in the import or the source itself, if you call it `ssd1306.SSD1306` it's always obvious what the thing is you're talking about. It may sound derivative, but I tend to look at adafruit for conventions in general, for example: `adafruit_displayio_sh1106.SH1106`. That is an intentional tautology, it is verbose, but not to the point where it looks like java. (That's why I dislike the "abstract" moniker for classes. It's pseudo-technical fluff that doesn't reflect what the word is used for in day-to-day conversations.)

Reply:
@xs5871 Good point. I think you're right that it would be better to be explicit in this case. I'll make sure to change back the names of the backend classes.

Also I agree it is probably better to give `DisplayBackend` a more descriptive name. Maybe something like `DisplayBackendBase`?

Reply:
We might as well drop the "Backend" in that case and just call it `DisplayBase`.

Reply:
The display backends now have their old names back and `DisplayBackend` is now `DisplayBase`